sdcm/cluster.py

```diff
@@ -3662,11 +3672,16 @@ def update_scylla_packages(node, _queue):
             node.remoter.run('tar -xvf /tmp/scylla/*.tar.gz -C /tmp/scylla/', ignore_status=True, verbose=True)
 
             # replace the packages
-            logging.info("Installing rpms")
-            node.remoter.run('yum list installed | grep scylla')
-            node.remoter.sudo('rpm -URvh --replacepkgs --replacefiles /tmp/scylla/*.rpm',
-                              ignore_status=False, verbose=True)
-            node.remoter.run('yum list installed | grep scylla')
+            if node.distro.is_rhel_like:
+                check_package_suites_distro(node, 'rpm')
```

Comment:
please add some logging.info messages.
I found it very helpful when had to debug issue around that.
Not always it's clear from the remoter.run output.